### ISSUE TITLE
PHP7 "use" statement with curly braces throws Notice: Uninitialized string offset: 0

### ISSUE THREAD

**Issue by drupalista-br** [opis/closure#23]
Declaring the "use" statements like below work just fine:
```
use App\SomeFolder\MyClass as Class1;
use App\SomeFolder\SomeSubfolder\MyClass as Class2;

```
Declaring them this way:
```
use App\SomeFolder\{
    MyClass as Class1,
    SomeSubfolder\MyClass as Class2,
};
```
Throws the following exception:
> In ReflectionClosure.php line 779:
>   [ErrorException]
>   Notice: Uninitialized string offset: 0
> Exception trace:
>  Opis\Closure\ReflectionClosure->fetchItems() at /my_project/vendor/opis/closure/src/ReflectionClosure.php:680
>  Opis\Closure\ReflectionClosure->getConstants() at /my_project/vendor/opis/closure/src/ReflectionClosure.php:472
>  Opis\Closure\ReflectionClosure->getCode() at /my_project/vendor/opis/closure/src/ReflectionClosure.php:559
>  Opis\Closure\ReflectionClosure->isBindingRequired() at /my_project/vendor/opis/closure/src/SerializableClosure.php:129
>  Opis\Closure\SerializableClosure->serialize() at n/a:n/a
>  serialize() at /my_project/src/my-script.php:41

**Comment by msarca** [comment]
This is because PHP 7.2 allow a trailing comma to the group-use. Removing the trailing comma after the `Class2` will give you a quick fix for this.

```php
use App\SomeFolder\{
    MyClass as Class1,
    SomeSubfolder\MyClass as Class2
};
```

**Comment by msarca** [comment]
Fixed in [v3.1.2](https://github.com/opis/closure/releases/tag/3.1.2)
